src/assets/svg/booster.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="42" height="40" viewBox="0 0 42 40" fill="none" xmlns="http://www.w3.org/2000/svg">

```

Comment:
I think it's suppose to be 40
```suggestion
<svg width="40" height="40" viewBox="0 0 42 40" fill="none" xmlns="http://www.w3.org/2000/svg">

```

Reply:
they should be `40 x 40` but we need to double check that the icon its still centrated

Reply:
the icon is being cut off 
![image](https://user-images.githubusercontent.com/12866949/116066552-7a25e900-a688-11eb-84ba-c41cd2723bdf.png)

Reply:
fixed